src/nav/full-stack-observability.yml

```diff
@@ -2583,13 +2583,15 @@ pages:
           - title: Enable Lambda monitoring
             path: /docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring
             pages:
-              - title: 1. Account linking
+              - title: Intro to enabling Lambda monitoring
+                path: /docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/enable-aws-lambda-monitoring
+              - title: Step 1. Account linking
                 path: /docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/account-linking
-              - title: 2. Instrumentation examples
+              - title: Step 2. Instrumentation examples
```

Comment:
@zuluecho9, how about recasting this so it uses a verb like this: "Step 2: Review examples"?